### DISCUSSION TITLE
Differentiate slider-end drops from combo breaks in scores

### DISCUSSION THREAD

**Discussion by velamyGH** [ppy/osu#24761] [Ideas] [answered]
essentially what i explained when i tweeted [this](https://twitter.com/garfeldkart/status/1697709676941427100), some sort of actual miss detection for things like medals would be nice because like i said it just seems a bit off to be able to play with sudden death and still not have the game detect a play as an "fc" as that is basically the purpose of the mod. Something like detecting the amount of 100s vs missing combo or again just a basic miss detection system would be a nice addition imo. I also agree with the post [here](https://github.com/ppy/osu/discussions/16998#discussioncomment-2316693) and think it would be a nice add :3

**Comment by 10kmotorola** [comment]
bump :>

**Comment by velamyGH** [comment]
bump, would still appreciate some official input regarding this now that medal solutions seem to be getting looked over at this time

**Reply by peppy** [reply]
slider end drops are shown on the results screen.

**Reply by velamyGH** [reply]
so could they be accounted against misses when awarding medals? sorry should have worded this better, a lot has changed since i wrote the op :')

**Reply by velamyGH** [reply]
in other words: medals (and other instances) where you have to fc should only look at misses and sliderbreaks, instead of the maximum combo and whether you get a "perfect combo" vs missing a slider end since this leads to a lot of scores that feel invalid to the game

**Reply by bdach** [reply, marked as answer]
Medals specifically are being reimplemented using stable logic as is with respect to "FCs". As in, you still need maximum combo for medals that required it on stable.

**Reply by velamyGH** [reply]
understandable, thanks for clarifying though :)
